profiles/kentik_snmp/velocloud/velocloud-edge.yml

```diff
@@ -32,3 +32,29 @@ metrics:
       OID: 1.3.6.1.2.1.25.3.3.1.2.196608
       poll_time_sec: 60
       tag: CPU
+  # VCE Link Name
+  - MIB: VELOCLOUD-MIB-EDGE             # The name of the target MIB
+    # The 'table' section is where you announce the target MIB table
+    table:
+      name: vceLink                         # The name of the target table
+      OID: 1.3.6.1.4.1.45346.1.1.2.2.2.3    # The root OID of the target table
+    symbols:
+      - name: VCE RxJitter
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.21
+      - name: VCE Link TxLatency
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.22  
+      - name: VCE Link RxLatency
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.23  
+      - name: VCE TxLostPkt
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.24
+      - name: VCE RxLostPkt
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.25
+      - name: VEC Link State
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.34
+    metric_tags:
+      - name: Velocloud Edge Link Name
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.3
+      - name: VCE Path Peer Name
+        OID: 1.3.6.1.4.1.45346.1.1.2.4.2.2.1.6
+      - name: VCE Link Itf
+        OID: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.33
```

Comment:
this is an enumerated OID; as it sits right now the only value returned will be an integer; you'll want to add the enumeration of `VceLinkStateType` from [the MIB file](https://jira.observium.org/secure/attachment/19423/VeloCloudEdge510-EnterpriseMib.txt) to make this human-readable.

```yaml
metric_tags:
  - tag: link_state
    column:
      name: vceLinkState
      oid: 1.3.6.1.4.1.45346.1.1.2.3.2.2.1.34
      enum:
        initial: 1
        dead: 2
        unusable: 3
        quiet: 4
        standby: 5
        unstable: 6
        stable: 7
        unknown: 8
```